frontend/packages/kubevirt-plugin/integration-tests/tests/dialogs/diskDialog.ts

```diff
@@ -72,9 +72,23 @@ export class DiskDialog {
   async edit(disk: StorageResource) {
     await this.fillName(disk.name);
     await this.selectInterface(disk.interface);
-    await this.fillSize(disk.size);
     await this.selectStorageClass(disk.storageClass);
+    if (disk.size) {
+      await this.fillSize(disk.size);
+    }
     await click(saveButton);
     await waitForNoLoaders();
   }
+
+  async add() {
+    await click(applyButton);
+  }
+
+  async save() {
```

Comment:
I am not big fan of these one-liner functions. IIUC, there is no expectation for additional business logic to be added here in the near future. Simple `    await click(saveButton);` makes more sense to me.

Similar comment to other functions.

Reply:
Done